### DISCUSSION TITLE
GitPython 3.1.30 for Python 3.6.x ?

### DISCUSSION THREAD

**Discussion by ykoer** [gitpython-developers/GitPython#1527] [Q&A] [answered]
Hello,
due to the [critical vulnerability](https://security.snyk.io/vuln/SNYK-PYTHON-GITPYTHON-3113858), Is there a chance to get GitPython 3.1.30 for Python 3.6.x ?

Thanks
Yusuf

**Comment by Byron** [comment, marked as answer]
I am not sure `GitPython` works for python 3.6 anymore as I believe that was removed a while ago due to EOL.
You might be able to backport the relevant fixes yourself though and use a fork.

This project doesn't support backporting and re-releaseing patches for previous versions either, and any improvements in this regard need to be contributed (like everything else).

**Reply by ykoer** [reply]
I agree. Thanks

**Reply by ykoer** [reply]
@Byron , I have one last question. 
Any idea why 3.1.30 is still classified as critical?
pyup.io-52322 (CVE-2022-24439)

**Reply by Byron** [reply]
<img width="940" alt="Screenshot 2023-01-06 at 07 20 20" src="https://user-images.githubusercontent.com/63622/210942370-4448db5b-a5e1-413e-a603-fea76b6dcde6.png">

To me it looks like they correctly classify v3.1.30 as patched, with the vulnerability fixed.

**Reply by ykoer** [reply]
Okay, it must be an issue with the advisory data in Quay/Clair
<img width="1186" alt="GitPython-3 1 30-Vulnerability" src="https://user-images.githubusercontent.com/3517764/210963207-18ec358b-b80d-4f0d-9535-d9c184601b8a.png">

Thanks a lot Byron
